src/components/TaskBody.vue

```diff
@@ -156,7 +156,8 @@ License along with this library. If not, see <http://www.gnu.org/licenses/>.
 					:trailing-button-label="subtasksCreationPlaceholder"
 					@trailing-button-click="addTask"
 					@keyup.esc="showSubtaskInput = false"
-					@keyup.enter="addTask">
+					@keyup.enter="addTask"
+					@paste="addMultipleTasks">
```

Comment:
Then we don't need to stop the event in the function.
```suggestion
					@paste.stop="addMultipleTasks">
```